### ISSUE TITLE
NEW SCENARIO: Chapeldown manor

### ISSUE THREAD

**Issue by antontimmermans** [NPBruce/valkyrie#1120]
### Description of Problem or Question
A haunted mansion for 2-5 players,
2 different layouts, a story line choosen out of 5 possibilities and many random events.

Requires SoA to play
gets more variaty with BtT 

### Valkyrie Version
2.2.03

### Attach Save, Quest or Scenario

https://github.com/antontimmermans/MOMCDM

**Comment by scrubbless** [comment]
Ohh more scenarios my pretties

**Comment by scrubbless** [comment]
@antontimmermans the link is dead. You repo says you only have two, your first scenario and 
Another one called momcdmold
![Screenshot_20190320-175643_Chrome](https://user-images.githubusercontent.com/32058089/54707754-b695cb00-4b39-11e9-9234-14817964c61f.jpg)

**Comment by antontimmermans** [comment]
sorry, it was marked as private, it is set public now
te MOMDATD is 'Disturbance at the Docks'
MOMCDMold is the work in progress version non-packaged

**MOMCDM** is the correct repository

**Comment by scrubbless** [comment]
Ok added the scenario. They don't appear right away in the new scenario browser.

**Comment by chrismaliszewski** [comment]
@antontimmermans, nice scenario! I don't have SoA nor BtT to play it so I only checked if there is everything all right with scripts/events. 

Would you consider changing the scenario so it'd be possible playing it with the basic game? :)
From what I see and if I'm right, it should be possible without much problem: changing a few tiles, characters' tokens and events (i.e. because of the statuses of players).

**Comment by antontimmermans** [comment]
@chrismaliszewski I will give it a try to make it based on base set only but it might be a bit more difficult then you think. I would love it for you to be able to play it, you have a good eye for detail and give nice feedback.

The puzzles and hints have references to the tiles. The person who helped me by testing the scenario also didn't have SoA, so he used other tiles instead and looked at the screen for the clues. Maybe for now you can try that too.

BTW, SoA is worthwhile to get it has the nicest added extensions and interesting scenarios in my opinion

**Comment by scrubbless** [comment]
I know you have some complex tile placement events so it may not be as easy as swapping a tile out and keeping the clues static.

Also I expect the clues in each room draw inspiration from the room itself? 

I did what @chrismaliszewski is asking with a stress and strain, removing the requirement of BtT. But I still couldn't completely back out RN due to base not having the correct themed tiles.

**Comment by scrubbless** [comment]
I've added Chapel Down Manor to the Stats Sheet too

**Comment by chrismaliszewski** [comment]
@antontimmermans, I'll try to give you feedback on antontimmermans/MOMCDM regarding my thoughts about changing some things to get rid of the necessity of having expansions soon. I'm travelling at the moment so I'll need to find a list and images of tiles of the base game.

My problem with playing MoM is that, unfortunately, I don't a good team to play the game :(. Since I bought the game ~9 months ago, I played more times alone (more than a dozen times) than with my wife (3-6 times) or with "my team" (3 times). Even worse, one person from the team plays only because others play; he doesn't pay attention to a plot, rushes things by clicking 'Continue' buttons without reading texts... 
Buying expansions might be a bad purchase for me at the moment since we haven't even played all scenarios from the basic game and I have a such a bad team :(.

**Comment by scrubbless** [comment]
@chrismaliszewski I am in the same situation, except I don't play solo.

I have played MoM with my wife quite a lot compared (maybe 12+ times) but my gaming group are not interested in MoM generally. We have played it ~6 times as a group, they don't find it techanically challanging enough. One of the gaming group is exactly the same as yours, skips text 'blah blah blah gain one clue', he actually did this during a play test of a stress and strain (which I found insulting since I have play test all of his games - since he writes RPGs). Anyway I tend to avoid MoM with my core gaming group, week to week.  It tends to hit the table when a couple of the guys that don't play often want to join, since it has an easy ruleset and is quick to setup (and they really enjoy it).
